docroot/modules/humsci/hs_layouts/hs_layouts.module

```diff
@@ -59,3 +59,18 @@ function hs_layouts_preprocess_pattern_spotlight(&$variables) {
     $variables['attributes']->addClass(Html::cleanCssIdentifier($bgValue));
   }
 }
+
+/**
+ * Implements hook_preprocess_preprocess_pattern_accordion().
+ */
+function hs_layouts_preprocess_pattern_accordion(&$variables) {
+  $paragraph = $variables['context']->getProperty('entity');
+  if (
+    $paragraph instanceof ParagraphInterface &&
+    $paragraph->hasField('field_hs_accord_expcol') &&
+    !$paragraph->field_hs_accord_expcol->isEmpty() &&
+    $paragraph->get('field_hs_accord_expcol')->first()->getValue()['value'] === '1'
```

Comment:
there's 2 possible ways to do this better. Since the field is just a boolean type, we don't really need to be so strict.
`$paragraph->get('field_hs_accord_expcol')->getString()`
or
`(bool) $paragraph->get('field_hs_accord_expcol')->getString()`